content/posts/2021/snakemake-support.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Snakemake Support"
```

Comment:
I'm sending proposed suggestions via attached diff file. The changes are as follows:
- simplifying introduction about Snakemake
- showing concrete Snakefile, useful for both old and new users
- reorderring the example a bit as a consequence, e.g. showing parameters early
- adding a section about hybrind workflows as an example how Snakemake users could profit from regular REANA goodies;
- changed the verbiage here and there.

Besides the suggested changes, it may be good to rename the file from `snakemake-support.md` to something like `support-for-running-snakemake-workflows.md` to go with an amended title.

Reply:
- Introduced the proposed changes.
- Added `<!--more-->` after the intro to have a simpler and nicer post preview on the home page.
- Renamed file.
- Added co-authors.

Check Netlify preview to verify.

Reply:
Just remembered that the file name becomes the URL, so maybe it should be simpler?

Reply:
I guess it's fine, the length seems acceptable `http://blog.reana.io/posts/2021/support-for-running-snakemake-workflows/` and we may be adding more Snakemake blog posts in the future, should this become popular... so it's good to keep the the URL indicating that this is a first blog post in the Snakemake series, as it were.